app/src/main/java/com/scrat/everchanging/Blick.java

```diff
@@ -81,7 +81,7 @@ final class Blick extends TextureObject {
     Blick(final Context context) {
         super(context, textureList, null);
         final FloatArraysReader floatArraysReader = new FloatArraysReader(context.getAssets());
-        matrixTransform = floatArraysReader.read(
+        matrixTransform = floatArraysReader.read3d(
                 "BlickMatrixTransform0",
                 "BlickMatrixTransform1"
```

Comment:
Actually when doing this I realised that we don't need multiple files to serialise 3d array, we can also make a format for 3d arrays with the same logic. I will try to migrate them into a single file in future PRs.